server.spec.js

```diff
@@ -9,9 +9,43 @@ const isSvg = require('is-svg')
 const path = require('path')
 const serverHelpers = require('./lib/in-process-server-test-helpers')
 const sinon = require('sinon')
+const Camp = require('camp')
 const svg2img = require('./gh-badges/lib/svg-to-img')
+const { handleRequest } = require('./lib/request-handler')
+const { loadServiceClasses } = require('./services')
 
 describe('The server', function() {
+  let dummyCamp = Camp.start({ port: config.port, hostname: '::' })
+  before('Check the services', function() {
+    // Without this block here, if registering a service fails, the log
+    // message is opaque:
+    //
+    // Error: listen EADDRINUSE :::1111
+    //   at Object._errnoException (util.js:1022:11)
+    //   at _exceptionWithHostPort (util.js:1044:20)
+    //   at Camp.setupListenHandle [as _listen2] (net.js:1367:14)
+    //   at listenInCluster (net.js:1408:12)
+    //   at doListen (net.js:1517:7)
+    //   at _combinedTickCallback (internal/process/next_tick.js:141:11)
+    //   at process._tickDomainCallback (internal/process/next_tick.js:218:9)
+    //
+    // Putting it here prevents the tests in this `describe()` from running at
+    // all.
```

Comment:
Change looks great, but I found this comment slightly confusing. Can we clarify that it prevents the tests in this `describe()` from running _if a service fails to register_

Reply:
Does that look better?